lib/readline.js

```diff
@@ -642,10 +652,10 @@ Interface.prototype.clearLine = function() {
 };
 
 
-Interface.prototype._line = function() {
+Interface.prototype._line = function(separator = '\n') {
```

Comment:
I suggest not to use a default value here and to be explicit in the call sites. Especially the call site that uses `s.split(/\r\n|\n|\r/)`. In that case it would be best to make sure these are kept as before as well.

Reply:
no problem, I'll revert the change. I was just being cautious about the bugs that might happen later if someone forgets to add the separator.